_episodes/01-introduction.md

```diff
@@ -118,49 +118,49 @@ Firstly we must decide what it is we want our Deep Learning system to do. Is it
 
 
-### 5. Choose a loss function and metrics
+### 5. Choose a loss function and optimizer
 
-The loss function tells the training algorithm how far away the predicted value was from the true value. There are a number of different types of function we can use here that might be suited to different types of data and network architectures. We'll look at choosing a loss function in more detail later on.
+The loss function tells the training algorithm how far away the predicted value was from the true value. We'll look at choosing a loss function in more detail later on.
 
+The optimizer is responsible for taking the output of the loss function and then applying some changes to the weights within the network. It is through this process that the "learning" (adjustment of the weights) is achieved.
 
-### 6. Train the model
 
-Before we can train the model we have one final component to decide upon, the optimizer. The optimizer is responsible for taking the output of the loss function and then applying some changes to weights within the network. We can think of the training process as like trying to gradually descend a (very jagged looking) curve on a graph (that looks more like a rocky mountain side than a smooth curve) and find the bottom of it, without knowing in advance where that bottom is. Along the way we might hit some flat or even rising parts of the curve that make it look like we've reached the bottom when we haven't, we call these local minima. Different optimizers and optimizer parameters can help prevent us getting stuck in local minima. As with the loss function the best choice may depend on the kind of data and network architecture that we're using.
+### 6. Train the model
 
-Now that we've set the optimizer function we can go ahead and start training our network. We'll probably keep doing this for a given number of iterations or _epochs_ or until the loss function gives a value under a certain threshold. The graphs below show the loss (generally) reducing as the number of epochs increases. Notice that on a few occasions it goes up instead of down. In the first graph it appears to have reached a plateau of near zero by the 500th epoch. The second graph shows the 500th to 1500th epoch and we can see that it is still going down, but frequently rises a little before dropping again.
+We can now go ahead and start training our neural network. We'll probably keep doing this for a given number of iterations or _epochs_ or until the loss function gives a value under a certain threshold. The graph below show the loss against the number of _epochs_, generally the loss will go down with each _epoch_, but occasionally it will see a small rise.
```

Comment:
```suggestion
We can now go ahead and start training our neural network. We'll probably keep doing this for a given number of iterations through our training dataset (referred to as _epochs_) or until the loss function gives a value under a certain threshold. The graph below show the loss against the number of _epochs_, generally the loss will go down with each _epoch_, but occasionally it will see a small rise.
```